tsconfig.json

```diff
@@ -1,15 +1,18 @@
 {
   "compilerOptions": {
-    "declaration": true,
+    "declaration": false,
```

Comment:
@aman19K Why are we disabling `declaration`. It should be enabled to generate `.d.ts` so that TypeScript can provide intellisense and accurate types.

Reply:
Added declaration:true.